docs/src/layouts/AppHeader.tsx

```diff
@@ -89,7 +103,7 @@ export default function AppHeader() {
             <IconButton
               component="a"
               color="primary"
-              href="https://github.com/mui"
+              href={`https://github.com/mui/${repositoryLink}`}
```

Comment:
Please no magic string concatenation

Reply:
So should I make a separate variable for this, with some comments explaining the two parts of the url(https://github.com/mui + 'repo')